openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+
+<!-- staff-info__student is necessary to get proper styling for the staff assessment form -->
+<div class="staff__grade__control ui-toggle-visibility is--collapsed staff-info__student">
+    <header class="staff__grade__header ui-toggle-visibility__control">
+        <h3 class="staff__grade__title">
+            <span class="wrapper--copy">
+                <span class="staff__grade__label">{% trans "Instructor Assessment" %}</span>
```

Comment:
We use "Staff Grade" and "Submit Assessment Grade Override". So I agree it makes sense to change it to "Staff Assessment".